apps/web/components/home/waitlist.tsx

```diff
@@ -102,12 +102,12 @@ export function WaitlistForm({ className }: WaitlistFormProps) {
 					onSubmit={handleSubmit(handleJoinWaitlist)}
 				>
 					<Input
-						placeholder="example@0.email"
+						placeholder="you@example.com" // Easier to understand
```

Comment:
0.email was a huge inspiration for this project. I'd like to pay some credit to them, though i understand your concern here. Same deal with email/password form
```suggestion
						placeholder="example@0.email"
```

Reply:
Noted